test/jasmine/tests/parcoords_test.js

```diff
@@ -0,0 +1,821 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var d3 = require('d3');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+var mock0 = { // mock with zero dimensions; special case, as no dimension can be rendered
```

Comment:
Ok sounds good. 

You're saying that image generation works outside of the docker container e.g. using `Plotly.toImage`? That's hard to believe without an `toSVG` method.